readthedocs/proxito/views/mixins.py

```diff
@@ -77,6 +98,11 @@ def _serve_docs_nginx(self, request, final_project, path):
         x_accel_redirect = iri_to_uri(path)
         response['X-Accel-Redirect'] = x_accel_redirect
 
+        if download:
+            filename_ext = urlparse(path).path.split('.')[-1]
```

Comment:
```suggestion
            filename_ext = urlparse(path).path.rsplit('.', 1)[-1]
```

This will probably never matter, but it's good to be defensive